### DISCUSSION TITLE
How to use JVM_XX_OPTS?

### DISCUSSION THREAD

**Discussion by alvarlagerlof** [itzg/docker-minecraft-server#1105] [Q&A] [answered]
There is no example. Correct formatting is not obvious, especially for multiple options.

**Comment by itzg** [comment, marked as answer]
Good point, that one is not clearly documented. Can you create an issue for that?

**Reply by itzg** [reply]
In the meantime it's the same space delimited list of arguments for the JVM as `JVM_OPTS`; however, the final JVM invocation references `JVM_XX_OPTS` first since the JVM requires certain ordering for those.

**Reply by itzg** [reply]
Since I was already in the README for the other discussion, I went ahead and clarified the syntax in https://github.com/itzg/docker-minecraft-server#jvm-options

**Reply by alvarlagerlof** [reply]
Alright, no issue then.
